packages/design-system/src/components/OcStatusIndicators/OcStatusIndicators.spec.ts

```diff
@@ -24,6 +24,9 @@ describe('OcStatusIndicators', () => {
         resource: fileResource,
         indicators: [indicator],
         target: 'test'
+      },
+      global: {
+        plugins: [...defaultPlugins()]
```

Comment:
why spreading? unnecessary `[Symbol.iterator]`call.

`defaultPlugins()`

Reply:
Basically all tests have it this way, just because it is easier to add more plugins. We think the overhead can be neglect in the test suite.